packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -108,13 +108,13 @@ class InnerBlocks extends Component {
 			renderAppender,
 			template,
 			__experimentalMoverDirection: moverDirection,
-			__experimentalTemplateOptions: templateOptions,
-			__experimentalOnSelectTemplateOption: onSelectTemplateOption,
-			__experimentalAllowTemplateOptionSkip: allowTemplateOptionSkip,
+			__experimentalPatterns: patterns,
+			__experimentalOnSelectPattern: onSelectPattern,
+			__experimentalAllowPatternSkip: allowPatternSkip,
```

Comment:
Why do we still need this InnerBlocks API. In my thinking, the introduction of the patterns API should remove any need for template/patterns behavior specific to InnerBlocks?

Reply:
I can look into it, this would simplify the `InnerBlocks` documentation and usage. @aduth any thoughts on this? It seems like with the refactor proposed, it wouldn't be a lot of work to use this component standalone.

Reply:
I removed all of them with f6b2034.